scripts/exportTypes.js

```diff
@@ -0,0 +1,93 @@
+/*!
```

Comment:
The license header here uses the old "All Rights Reserved" style - should it have a date of 2014?  Or is this new code?